notebooks/lesson07_classification.ipynb

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
<p>I don't know if you plan to do this in the slides, but I think it would be good to underline the difference between regression and classification again. maybe also state again what the leaf in a regression and classification forest yield. just because last year we had some confusion about this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>yeah i'll do this on the ipad during the lesson</p>